app/components/PhotographerCard.js

```diff
@@ -0,0 +1,203 @@
+'use client';
+
+import { useState } from 'react';
+import Image from 'next/image';
+import Card from '@mui/material/Card';
+import CardActionArea from '@mui/material/CardActionArea';
+import CardActions from '@mui/material/CardActions';
+import CardContent from '@mui/material/CardContent';
+import Typography from '@mui/material/Typography';
+import Chip from '@mui/material/Chip';
+import Stack from '@mui/material/Stack';
+import Box from '@mui/material/Box';
+import Button from '@mui/material/Button';
+import StarIcon from '@mui/icons-material/star';
+import FavoriteIcon from '@mui/icons-material/Favorite';
+import { Noto_Sans_TC } from 'next/font/google';
+import * as theme from '@/app/lib/theme'
+
+const black_card_text = theme.gradient.black_card_text
+
+const noto_sans_tc = Noto_Sans_TC({
+  subsets: ['latin'],
+  weight: ['400', '700'],
+});
+
+export default function PhotographerCard({
+  name,
+  location,
+  rate,
+  catagories,
+  price,
+  img_url
+}) {
+  const theme = {
+    secondary: 'rgba(234, 80, 123, 1)',
+  };
+  const [isFavorite, setIsFavorite] = useState(false);
+  function addFavorite() {
+    setIsFavorite(!isFavorite);
+  }
+
+  return (
+    <Card
+      sx={{
+        position: 'relative',
+      }}
+    >
+      <Image
+        alt={img_url}
+        src={img_url}
+        width={319}
+        height={424}
+        style={{
+          width: '100%',
+          height: 'auto',
+          aspectRatio: '3 / 4',
+          objectFit: 'cover',
+          display: 'block',
+        }}
+        priority={true}
+      />
+       
+          <CardContent
+        sx={{
+          padding: 0,
+          position: 'absolute',
+          top: 0,
+          bottom: 0,
+          left: 0,
+          right: 0,
+          zIndex: 1,
+        }}
+      >
+        <Box
+          sx={{
+            display: 'flex',
+            justifyContent: 'space-between',
+            padding: 2,
+            position: 'absolute',
+            zIndex: 2
+          }}
+        >
+          <Typography
+            sx={{
+              color: 'rgba(31, 19, 0, 1)',
```

Comment:
這裡設計稿用黑色，總覺得哪裡怪怪的，感覺白色好一些（？

```suggestion
              color: 'primary.main',
```

Reply:
那我這邊先改為白色~